static/src/javascripts/bootstraps/sport.js

```diff
@@ -57,6 +59,31 @@ define([
                         $('.js-football-tabs').append(nav);
                     }
                 });
+
+                var content = $.create(resp.scoreEvents);
+                var dropdownTemplate = detect.isBreakpoint({ max: 'mobile' }) && resp.dropdown;
+                var name = 'Score breakdown';
+                var button;
+
+                if (detect.isBreakpoint({ max: 'mobile' })) {
+                    $('.js-after-article').append(
+                        $.create(dropdownTemplate).each(function (dropdown) {
+                            var $dropdown = $(dropdown).addClass('dropdown--key-events');
+                            if (config.page.isLiveBlog) { $dropdown.addClass('dropdown--key-events'); }
+                            $dropdown.addClass('dropdown--active');
+                            $('.dropdown__label', dropdown).append(name);
+                            $('.dropdown__content', dropdown).append(content);
+                            button = $('.dropdown__button', dropdown);
+                            button
+                                .attr('data-link-name', 'Show dropdown: ' + name);
+                        })
+                    );
+                } else {
+                    var placeholder = $.create('<div class="hide-on-mobile"></div>');
+                    $('.score-container').after(placeholder);
+                    placeholder.append(content);
+                }
+
```

Comment:
This isn't nice—we stole this from football—but it does the job. We are hijacking the component that renders live scores to also render the match nav and now score events. This is so we don't make extra requests. The component paradigm doesn't play well when you have multiple components for a single request. If I had more time I would have ditched the component architecture.